### PR TITLE
Update README to point to nio as an alternative library

### PR DESCRIPTION
**Pull request by non-Jedi**
Signed-off-by: Adam Beckmeyer <adam_gpg@thebeckmeyers.xyz>

### CODE REVIEW

**Comment by Half-Shot** [comment]
Looks good, wondering if we could make it even more prominent?

**Comment by non-Jedi** [comment]
Good idea. I'll put it at the top of that section and use more forceful language. Something like "We strongly recommend using matrix-nio rather than this project".

**Comment by non-Jedi** [comment]
Better?

File: README.rst [hunk 1]
```diff
@@ -10,26 +10,31 @@ Matrix Client SDK for Python
...
-For more info or to volunteer to help, please see
-https://github.com/matrix-org/matrix-python-sdk/issues/279. Or come chat in
-`#matrix-python-sdk:matrix.org`_.
+We strongly recommend using the `matrix-nio`_ library rather than this
+sdk. It is both more featureful and more actively maintained.
+
+This sdk is currently lightly maintained without any person ultimately
+responsible for the project. Pull-requests *might* be reviewed but no
+new-feature or bug-fix development is happening outside of pull
+requests.  For more info or to volunteer to help, please see
```
Comment (Half-Shot) [inline README.rst]: This reads a little funny. Perhaps:

"Pull requests *may* be reviewed in certain circumstances but no new features or bug fixes are being actively developed."

Comment (non-Jedi) [reply README.rst]: fine by me

**Pull request merged**